odxtools/multiplexer.py

```diff
@@ -76,9 +76,15 @@ class MultiplexerSwitchKey:
     """This class represents a Switch Key, which is used to select one of the cases defined in the Multiplexer."""
 
     byte_position: int
-    bit_position: int
+    bit_position: Optional[int]
```

Comment:
Default bit_position is 0 for multiplexer
```suggestion
    bit_position: int
```

Reply:
according to the XSD, it isn't:

![image](https://user-images.githubusercontent.com/1837735/198320905-774ea439-4fa5-41ba-ba0a-a1380d801613.png)

maybe I made a mistake, though...

Reply:
For switch key, a no bit position is equivalent to a bit position of 0, the same logic does not apply to parameters

Reply:
yep, but it should still be possible to distinguish between "BIT-POSITION" not specified and it being 0 for writing the database back into a PDX file. also, it would be pretty confusing to handle it differently than the parameters...